arduino/libify.sh

```diff
@@ -118,20 +161,22 @@ function copy_dir() {
     pushd ${TARGET_LIB_DIR}/$1 >/dev/null
     
     if [ "x$VERBOSE" != "x" ]; then
-        echo "${OPENMRNPATH}/${2} ==> ${TARGET_LIB_DIR}/$1"
+        echo "${INVERSE_DIR}${OPENMRNPATH}/${2} ==> ${TARGET_LIB_DIR}/$1"
     fi
 
     if [[ "$OSTYPE" == "darwin"* ]]; then
-        cp -fa ${USE_LINK} ${OPENMRNPATH}/$2 .
+        cp -fa ${USE_LINK} ${INVERSE_DIR}${OPENMRNPATH}/$2 .
     else
-        cp -faxr ${USE_LINK} ${OPENMRNPATH}/$2 .
+        cp -faxr ${USE_LINK} ${INVERSE_DIR}${OPENMRNPATH}/$2 .
     fi
 
     popd >/dev/null
 }
 
-copy_file . arduino/{library.json,library.properties,keywords.txt,README.md,LICENSE,CONTRIBUTING.md}
-copy_dir . arduino/examples
+if [ "x$TARGET_IDF" == "x" ]; then
+    copy_file . arduino/{library.json,library.properties,keywords.txt,README.md,LICENSE,CONTRIBUTING.md}
```

Comment:
we should probably have `README.md,LICENSE,CONTRIBUTING.md` on the IDF side, perhaps a slightly different version though.

Reply:
added

Reply:
I didn't add contributing.md because you don't have it, and the arduino version references the OpenMRNLite library in the text. So we'd need to first write a new one, but this is really not high priority now.